frontend/app/components/limber/layout/orientation.gts

```diff
@@ -1,16 +1,15 @@
 // @ts-ignore
 import { hash } from '@ember/helper';
 
-import ContainerQuery from 'ember-container-query/components/container-query';
-import aspectRatio from 'ember-container-query/helpers/cq-aspect-ratio';
+import { ContainerQuery, aspectRatio } from 'ember-container-query';
 
 import constrainVertically from 'limber/modifiers/constrain-vertically';
 
 import type { TOC } from '@ember/component/template-only';
 
 interface Signature {
   Blocks: {
```

Comment:
I learned that the type yielded from ember-container-query, Metadata, is wrong (should be boolean, as revealed via `{{log query.features}}`, so more updates to the patchfile are needed